<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>960</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="Widget_3D">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>951</width>
      <height>921</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
     background-color: #000000;
     font-family: Rubik;
     color: white;
     font-size: 16pt;
     border: none; /* Добавлено свойство border: none; */
   }

   QPushButton#btn_jpeg, QPushButton#btn_bmp,
   QPushButton#btn_gif, QLineEdit#name_file {
     background: #cccccc;
     color: #000000;
     border-radius: 20%;
     font-size: 18px;
     min-width: 40px;
     min-height: 40px;
   }

   QPushButton#btn_file, QPushButton#btn_scalate,
   QPushButton#btn_settings, QPushButton#btn_translate,
   QPushButton#btn_rotate
   {
     background: #ff6600;
     color: #ffffff;
     border-radius: 20%;
     font-size: 18px;
     min-width: 40px;
     min-height: 40px;
   }

   QPushButton#btn_jpeg:hover,
   QPushButton#btn_bmp:hover,
   QPushButton#btn_gif:hover,
   QPushButton#name_file:hover
   {
   background-color: #666;
   }

   QPushButton#btn_file:hover,
   QPushButton#btn_scalate:hover,
   QPushButton#btn_settings:hover,
   QPushButton#btn_translate:hover,
   QPushButton#btn_rotate:hover
   {
   background: #e7a477;
   color: #ffffff;
   }

   QPushButton#btn_file:pressed,
   QPushButton#btn_scalate:pressed,
   QPushButton#btn_settings:pressed,
   QPushButton#btn_translate:pressed,
   QPushButton#btn_rotate:pressed
   {
   background: #ffffff;
   color: #ff6600;
   }

   QPushButton#btn_jpeg:pressed,
   QPushButton#btn_bmp:pressed,
   QPushButton#btn_gif:pressed,
   QPushButton#name_file:pressed
   {
   background: #b6b6b6;
   color: #ffffff;
   }
   QGraphicsView {
        background-color: rgb(60, 60, 60);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_zoom">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>90</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_proection_tipe">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>490</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Projection type:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_jpeg">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>890</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>jpeg save</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_line_tipe">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>570</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Line paint type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_vershini_tipe">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>570</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertexes paint type:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bmp">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>890</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>bmp save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_color_screen">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>840</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Background color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gif">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>890</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>gif save</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_scale_x">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="zoom_slider">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>120</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_scale">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>440</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Set 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_scale">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>440</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_move_z">
    <property name="geometry">
     <rect>
      <x>981</x>
      <y>310</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_move_y">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>390</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_x">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_y">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_move_z">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>390</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_file">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_move_x">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>350</y>
      <width>20</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_file_name">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>40</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Name obj:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rotate_x">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>180</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>x°:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rotate_y">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>220</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>y°:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rotate_z">
    <property name="geometry">
     <rect>
      <x>981</x>
      <y>260</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>z°:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_x">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_y">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_rotate_z">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>5.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rotate">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>180</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_rotate">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>180</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Set 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_move">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>310</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_move">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>310</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Set 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_line_tolshina">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>690</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Line size:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_line_tolshina">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>710</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_color_line">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>800</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Line color</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_vershini_size">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>690</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertexes size:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_vershini_size">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>710</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_color_vershini">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>760</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertexes color</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>510</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="btn_radio_parallel">
       <property name="text">
        <string>Parallel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn_radio_central">
       <property name="text">
        <string>Central</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>590</y>
      <width>131</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="btn_radio_cube">
       <property name="text">
        <string>Square</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn_radio_kryg">
       <property name="text">
        <string>Round</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn_radio_nan">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>590</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QRadioButton" name="btn_radio_sploshnaya">
       <property name="text">
        <string>Solid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn_radio_punktir">
       <property name="text">
        <string>Dotted</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
